bigbluebutton-html5/imports/ui/components/presentation/download-presentation-button/styles.scss

```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-transparent: #ff000000;
+  ::-webkit-media-controls {
```

Comment:
The sass compiler don't add the prefix? if not, should add to firefox `-moz-` and IE/edge`-ms-`.

Reply:
To be honest I copied this from the fullscreen component and am not sure if it's needed and if it should be improved as you suggested. Would appreciate some input from @pedrobmarin